### ISSUE TITLE
Vertical Splits not working (in neovim and vim)

### ISSUE THREAD

**Issue by proever** [facebook/PathPicker#279]
Hi there, I can't seem to get fpp to open my selected files in splits. Instead they open for sequential editing. I've set the FPP_EDITOR variable in my bash_profile to use nvim. 

I'm not quite sure why it's not working to be honest. I've had a look at src/output.py, line 114. I can manually generate the command that would be generated there and it seems to work (although nvim -O might be a simpler way of doing it).

**Comment by pcottle** [comment]
Can you try catting out what fpp generates? That should help us debug.

It might be nvim not respecting the -O option?

```
[pcottle:~:]$ cat ~/.fpp/.fpp.sh 

vim -O './visuals/visEdge.js' './visuals/visTag.js'
exit;
```

**Comment by proever** [comment]
Sure thing. Here's what I get: 

05:08:18 : Pauls-MacBook-Pro > proever > dotfiles $ cat ~/.fpp/.fpp.sh

nvim  './bash_profile.sh' './init.vim' './tmux.conf'
exit $?;

I think I've just realized what's happening though. The release I'm on (0.7.2), which I got from homebrew, doesn't seem to include [https://github.com/facebook/PathPicker/commit/1e36e79ca2be10e0073771bcfebad6b586075c49](url) yet, which is why it's not working. 

I can confirm that manually installing it solves the issue.
